src/components/Button.js

```diff
@@ -1,14 +1,24 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
-const Button = ({ buttonName }) => <button type="button">{buttonName}</button>;
-
+const Button = ({ buttonName, color, wide }) => {
+  const twice = wide === true && 'wide';
+  return (
+    <button type="button" className={`button border ${twice} ${color}`}>
+      {buttonName}
+    </button>
+  );
+};
 Button.propTypes = {
   buttonName: PropTypes.string,
+  color: PropTypes.string,
+  wide: PropTypes.bool,
 };
 
 Button.defaultProps = {
   buttonName: '',
```

Comment:
Please make `buttonName` a required prop. It shouldn't have an empty string as a default prop.